src/dbnode/persist/fs/reader_open_options_matcher.go

```diff
@@ -48,6 +48,9 @@ func (m ReaderOpenOptionsMatcher) Matches(x interface{}) bool {
 	if !m.ID.BlockStart.Equal(readerOpenOptions.Identifier.BlockStart) {
 		return false
 	}
+	if m.ID.VolumeIndex != (readerOpenOptions.Identifier.VolumeIndex) {
```

Comment:
nit: Don't need the brackets on the right hand side evaluation here.

Reply:
Lol I think that was just copy pasta from the check right above it. Fixed